update data formats according information from community.
(it should be propagated to CLDR project probably)

--- cldr_core.orig/common/main/bn.xml
+++ cldr_core/common/main/bn.xml
@@ -1596,12 +1596,12 @@
 				<dateFormats>
 					<dateFormatLength type="full">
 						<dateFormat>
-							<pattern>EEEE, d MMMM, y</pattern>
+ 							<pattern>EEEE, yyyy MMMM dd</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="long">
 						<dateFormat>
-							<pattern>d MMMM, y</pattern>
+ 							<pattern>EEEE MMMM dd yyyy</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="medium">
@@ -1611,7 +1611,7 @@
 					</dateFormatLength>
 					<dateFormatLength type="short">
 						<dateFormat>
-							<pattern>d/M/yy</pattern>
+ 							<pattern>dd-M-yyyy</pattern>
 						</dateFormat>
 					</dateFormatLength>
 				</dateFormats>
